mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -40,6 +59,16 @@ odoo.define('mail_tracking.partner_tracking', function(require){
             return !!(this._partnerTrackings && (this._partnerTrackings.length > 0));
         },
 
+        /**
+         * State whether this message contains some email Cc values
+         *
+         * @override
+         * @return {boolean}
+         */
+        hasEmailCc: function () {
+            return !!(this._emailCc && (this._emailCc.length > 0));
```

Comment:
Remove some unneeded syntax:

```suggestion
            return this._emailCc && this._emailCc.length > 0;
```